### ISSUE TITLE
Docs issue report from "features/collaboration/comments/comments-archive.html"

### ISSUE THREAD

**Issue by ryanj-openquire** [ckeditor/ckeditor5#17198]
### Origin URL

https://ckeditor.com/docs/ckeditor5/latest/features/collaboration/comments/comments-archive.html

### Project version

43.1.1

### Is the information outdated? How?

I tried to follow the example, and when I ran the code I received an exception that the "CommentsArchiveUI" plugin was not loaded. In fact the CommentsArchiveUI plugin class is not exported out of the comments package:
export 'CommentsArchiveUI' (imported as 'CommentsArchiveUI') was not found in '@ckeditor/ckeditor5-comments' (possible exports: Annotation, AnnotationCollection, AnnotationView, Annotations, AnnotationsUIs, BaseCommentThreadView, BaseCommentView, Comment, CommentThread, CommentThreadController, CommentThreadInputView, CommentThreadView, CommentView, Comments, CommentsArchive, CommentsListView, CommentsOnly, CommentsRepository, CommentsUI, EditorAnnotations, InlineAnnotations, NarrowSidebar, WideSidebar)

### Is there something missing in the guide? What is it?

How are we supposed to access the CommentsArchiveUI?

### User agent

Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:130.0) Gecko/20100101 Firefox/130.0

**Comment by ryanj-openquire** [comment]
After debugging this further, I see what the problem is in our use case. In our integration, there are times where we create an Editor instance, and other times we create a Context. We have created wrappers around these 2 types, such that the rest of our code is written in a way that it doesn't care. However, as part of initializing either one of these, we were setting the "extraPlugins" config property. Just discovered that only the Editor honors this property, while the Context ignores it. That means the core Comments plugin that we specified in extraPlugins is not being loaded by the Context. It would be great if Context also supported the extraPlugins config property. We'll find a different way of solving this in the mean time
